routes/web.php

```diff
@@ -17,6 +17,7 @@
 
 Auth::routes();
 
+Route::get('/users', 'UsersController@index')->name('users.index');
 Route::get('/users/{user}', 'UsersController@show')->name('users.show');
```

Comment:
you can use the group function of routes so that it will be understandable the dependencies of each routes.

Reply:
noted. thank you.